tests/robot-tests/tests/libs/admin/manage-content-common.robot

```diff
@@ -216,6 +216,7 @@ user deletes editable accordion section
     user clicks button    Remove this section    ${section}
     user waits until modal is visible    Are you sure?
     user clicks button    Confirm
+    user waits until page does not contain button    Confirm
```

Comment:
I'd be tempted to rename this `user waits until modal is not visible  Are you sure?` or alternatively just `user waits util modal is not visible`, as we only ever normally deal with one at a time.

Then it can be used across all tests that use the keyword `user waits until modal is visible` to do the corresponding closed test.